packages/devtools/lib/src/memory/memory_controller.dart

```diff
@@ -148,3 +183,177 @@ class MemoryController {
     return false;
   }
 }
+
+class Settings {
+  String pattern = '*';
+  bool showPrivateFields = true;
+  bool experiment = false;
+}
+
+const String _dartLibraryUriPrefix = 'dart:';
+const String _flutterLibraryUriPrefix = 'package:flutter';
+
+class FilteredLibraries {
+  final List<String> _filteredLibraries = [
+    normalizedDartLibraryUri,
+    normalizedFlutterLibraryUri,
+  ];
+
+  static const String normalizedDartLibraryUri = 'Dart';
+  static const String normalizedFlutterLibraryUri = 'Flutter';
+
+  static String normalizeLibraryUri(Library library) {
+    final uriParts = library.uri.split('/');
+    final firstPart = uriParts[0];
```

Comment:
nit
urlParts.first instead of urlParts[0]

Reply:
Done.